src/hotspot/share/compiler/oopMap.cpp

```diff
@@ -29,6 +29,7 @@
 #include "code/scopeDesc.hpp"
 #include "compiler/oopMap.hpp"
 #include "gc/shared/collectedHeap.hpp"
+#include "jvmci/jvmci_globals.hpp"
```

Comment:
Also #if INCLUDE_JVMCI here?

Reply:
Thanks for pointing this out. Agree. Will add soon.